### ISSUE TITLE
Err:6 http://ppa.launchpad.net/kisak/kisak-mesa/ubuntu jammy Release, 404  Not Found [IP: 2001:67c:1560:8008::19 80], E: The repository 'http://ppa.launchpad.net/kisak/kisak-mesa/ubuntu jammy Release' does not have a Release file. (on a fresh install of MX Linux 21 KDE from the Final sourceforge directory)

### ISSUE THREAD

**Issue by id-8086** [lutris/docs#80]
I get the following (`after sudo add-apt-repository ppa:kisak/kisak-mesa && sudo dpkg --add-architecture i386 && sudo apt update && sudo apt upgrade && sudo apt install libgl1-mesa-dri:i386 mesa-vulkan-drivers mesa-vulkan-drivers:i386`):
```

 The goal of this PPA is to provide the latest point release of Mesa plus select non-invasive early backports. Deviations from upstream packages are listed on the package details page.

--- Support status ---

Bionic (18.04) - Supported (Please use the HWE X and kernel or equivalent)
Focal (20.04) - Supported
Hirsute (21.04) - Supported
Impish (21.10) - Preliminary Support (not tested locally)

--- Is this PPA stable? ---

Short answer: Mostly.

Long answer: Compared to bleeding edge mesa PPAs, there is a much lower chance of complications, but as Ubuntu LTS releases age, the odds of unexpected interactions between older kernels / compilers / X increases with each major release. Anomalies are inevitable the further the host system gets from the configuration active mesa developers use on a daily basis. If you value stability over support, https://launchpad.net/~kisak/+archive/ubuntu/turtle/ is available as an alternative.

In the event that there is a major issue with new mesa on an older Ubuntu LTS and the mesa devs are not interested in triaging the issue, then the Ubuntu LTS release will be dropped from this PPA and the last release pushed to kisak-mesa stable will be frozen.

--- Package status ---

llvm - Following latest point release supported by mesa (On hold - blocked by build farm i386 whitelist)
mesa - Following latest point release
libclc - Obsolete, now provided by llvm packaging
libdrm - Updating as needed for mesa

--- Removal ---

It's strongly recommended to remove this PPA before upgrading to a newer Ubuntu release or using another mesa PPA.

sudo apt install ppa-purge
sudo ppa-purge ppa:kisak/kisak-mesa

Note: Using ppa-purge with Ubuntu derivatives needs to include -d <based_on_name> to work safely. For example, Linux Mint 20 is based on Ubuntu Focal, so that would make it:

sudo ppa-purge -d focal ppa:kisak/kisak-mesa

--- Donations ---

I can't accept donations and any random donation site account is a scam. If you have some extra money burning a hole in your pocket, please consider sending it to a charity of your choice (for the poors, animals, whatever else you may think it might need it), then send Oibaf a note that I stole the suggestion from his PPA.
 More info: https://launchpad.net/~kisak/+archive/ubuntu/kisak-mesa
Press [ENTER] to continue or ctrl-c to cancel adding it

gpg: keybox '/tmp/tmp5fwtlrv_/pubring.gpg' created
gpg: /tmp/tmp5fwtlrv_/trustdb.gpg: trustdb created
gpg: key F63F0F2B90935439: public key "Launchpad PPA for kisak" imported
gpg: Total number processed: 1
gpg:               imported: 1
Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).
gpg: no valid OpenPGP data found.
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.9/threading.py", line 892, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/softwareproperties/SoftwareProperties.py", line 688, in addkey_func
    func(**kwargs)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 386, in add_key
    return apsk.add_ppa_signing_key()
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 273, in add_ppa_signing_key
    cleanup(tmp_keyring_dir)
  File "/usr/lib/python3/dist-packages/softwareproperties/ppa.py", line 234, in cleanup
    shutil.rmtree(tmp_keyring_dir)
  File "/usr/lib/python3.9/shutil.py", line 718, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/usr/lib/python3.9/shutil.py", line 675, in _rmtree_safe_fd
    onerror(os.unlink, fullname, sys.exc_info())
  File "/usr/lib/python3.9/shutil.py", line 673, in _rmtree_safe_fd
    os.unlink(entry.name, dir_fd=topfd)
FileNotFoundError: [Errno 2] No such file or directory: 'S.gpg-agent.browser'
Hit:1 https://espejito.fder.edu.uy/mxlinux/packages/mx/repo bullseye InRelease
Hit:2 http://deb.debian.org/debian bullseye-updates InRelease              
Hit:3 http://security.debian.org/debian-security bullseye-security InRelease
Ign:4 http://ppa.launchpad.net/kisak/kisak-mesa/ubuntu jammy InRelease
Hit:5 http://deb.debian.org/debian bullseye InRelease                      
Err:6 http://ppa.launchpad.net/kisak/kisak-mesa/ubuntu jammy Release       
  404  Not Found [IP: 2001:67c:1560:8008::19 80]
Reading package lists... Done                                
E: The repository 'http://ppa.launchpad.net/kisak/kisak-mesa/ubuntu jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
```

**Comment by tannisroot** [comment]
Use a non-development version of Ubuntu (not Jammy). I don't think any PPAs provide a jammy release yet.

**Comment by tannisroot** [comment]
Oh, it's not Ubuntu. Yeah just don't use MX Linux, weird distro if it identifies as a development version of Ubuntu.
I suggest Pop OS.

**Comment by id-8086** [comment]
> Oh, it's not Ubuntu. Yeah just don't use MX Linux, weird distro if it identifies as a development version of Ubuntu. I suggest Pop OS.

Is your suggestion serious?

**Comment by Satook** [comment]
Jammy is Ubuntu 20.04 and anything derived from it. I'm not using Lutris right now but have the same issue getting the latest mesa from Kisak via the PPAs. There's no Jammy build just yet. Should pop up in a few days/weeks.

**Comment by id-8086** [comment]
Have a nice day.

------- Original Message -------

**Comment by id-8086** [comment]
You too :)

------- Original Message -------
